CONTRIBUTING.md

```diff
@@ -226,6 +227,21 @@ If the PR passes CI tests and does not have any `need:` labels, it is ready for
 
 If an issue/PR has been inactive for >=30 days, it will be closed. If it does not have any status label, add `inactive`.
 
+## Revert Guidelines
+
+Reverting changes that are causing issues in the master branch is a normal and expected part of the development process. In an open source community, the ramifications of a change cannot always be fully understood. With that in mind, here are some considerations to keep in mind when considering a revert:
```

Comment:
Do you feel like age of the PR is something to consider? I'd say recent PRs are more acceptable to revert than very old ones for a couple reasons: it means the regression was discovered quickly so it likely affecting multiple users and it's unlikely the revert will cause issues due to other logic committed after the bad PR.

Reply:
Good point. Will add.